main.pm

```diff
@@ -92,6 +92,10 @@ sub cleanup_needles() {
 
 }
 
+sub is_staging () {
+    return get_var('STAGING');
```

Comment:
we've using this check in many place in main.pm then we don't need to change product settings ie. don't need to rebuild all staging iso for now